src/components/ui/Links/Links.js

```diff
@@ -1,13 +1,10 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Paragraph } from '../Paragraph/Paragraph';
 
-export const Links = ({ children, to }) => {
+export const Links = ({ children, to, className }) => {
```

Comment:
also rename this component to singular, it was confusing to me reading this

rename to link